### DISCUSSION TITLE
Allow custom icons for floating buttons

### DISCUSSION THREAD

**Discussion by spasauna** [elementor/elementor#28741] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm always frustrated when I see the perfect floating buttons widget but I can't customize the icons for some unknown reason.

### Describe the solution you'd like.

Allow custom icons to be uploaded and selected from the media library and icon packs as usual.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by wdautreix** [comment]
It would be very usefull to be able to modify the colors / icons / and links of **each** floating CTA ! As it stands, it is unusable. Unless you want to do a simple / copy and paste of the model.

**Reply by spasauna** [reply]
I totally agree! Changing colours etc. could be done with CSS but why make it so unfriendly for us users to make such easy changes? Changing the icons etc. is probably a bit more complicated than that.

**Comment by sflwa** [comment]
Adding to this
Give the ability to have a "custom" which allows you to put in a custom URL / link as well as select the icon as this could be used for links to PDFs / social media / other systems

**Comment by mwatkins197** [comment]
I agree with @sflwa this is an important feature which is missing.  I would like to link to an online meeting or quote form, but currently you cannot change the icon of the url in the action areas.

**Comment by XVII** [comment]
It's odd -- we used to have a floating action button that I'm sure was mostly custom (icon, URL, etc), but in a recent update it's just gone?

I just want a simple, single FAB with a custom icon & URL to specific location.

**Comment by minikate** [comment]
Yes please to this.

**Comment by madancodewing** [comment]
Currently, the Elementor Button widget does not provide separate styling options for the icon (such as icon size and icon color). The icon inherits the text styles, which limits the design flexibility we can achieve.

Please consider adding dedicated controls to customize the icon size, icon color, and optionally icon spacing independently from the text styles in the Button widget. This would allow for more advanced and modern design variations.

Why it's useful:
- Allows icons to stand out with unique colors and sizes.
- Helps match button designs with branding and UI requirements.
- Reduces the need for custom CSS for what should be basic design options.

**Comment by acal** [comment]
+1000000 this is the main reason I do not use floating elements. Far too rigid. I need the full icon library and even SVG uploads for it to be practical. Additional styling tools a plus.

**Comment by parfilov** [comment]
As usual, they released a feature and abandoned it. Of course, you can't hype up an old feature and make any new money.
